gateway/src/main/resources/templates/error/501.html

```diff
@@ -0,0 +1,46 @@
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta http-equiv="cache-control" content="no-cache" />
+    <meta http-equiv="pragma" content="no-cache" />
+    <meta name="robots" content="none" />
+    <meta name="googlebot" content="noarchive" />
+    <title>Site en maintenance</title>
```

Comment:
Could you please write the sentence in english instead of french?

Reply:
done